Suite2p/suite2p_parameters.py

```diff
@@ -0,0 +1,70 @@
+#import os
+import sys
+#import inspect
+import h5py
+#import numpy as np
+
+sys.path.append("..")
+import utilities as utils
+import definitions as defs
+import suite2p_definitions as s2p_defs
+
+import suite2p
+
+class Suite2pParameters:
+    """
+    Parameters used in suite2p library
+    """
+
+    def __init__(self, danse_params: dict):
+        self.paths: dict          = danse_params.get(defs.Parameters.Main.PATHS, {})
+        self.params: dict         = danse_params.get(defs.Parameters.Main.PARAMETERS, {})
+        self.preview_params: dict = danse_params.get(defs.Parameters.Main.PREVIEW, {})
+        self.timeLength: int      = self.get_time_length()
+        
+        self.ops = suite2p.default_ops()
+        self.ops['batch_size'] = 50 # we will decrease the batch_size in case low RAM on computer
```

Comment:
Are those your comments or the comments from Suite2p example code?

Reply:
It was from suite2p example.
We changed it

Reply:
Could you please point me in what places exactly it's not the same?
I'm going to add Suite2p for microscope data by myself.